Weather_App/Weather_App/Scenes/Map/MapViewModel.swift

```diff
@@ -131,6 +133,14 @@ extension MapViewModel: ViewModelType {
             })
             .disposed(by: disposeBag)
         
+        input.getDetailWeatherTrigger
+            .withLatestFrom(weatherCurrentData)
+            .compactMap { $0 }
+            .drive(onNext: {
+                navigator.toDetailViewController(weatherCurrentEntity: $0)
+            })
```

Comment:
toMapDetail hay 1 cái tên gì đó khác đi e
```suggestion
            .drive(onNext: navigator.toDetailViewController)
```